content/en/docs/concepts/security/index.md

```diff
@@ -630,333 +631,277 @@ runtime. When a request comes to the proxy, the authorization engine evaluates
-object is a cluster-scoped singleton with a fixed name value of `default`. You can
-only use one `ClusterRbacConfig` instance in the mesh. Like other Istio configuration
-objects, `ClusterRbacConfig` is defined as a
-Kubernetes `CustomResourceDefinition`
-[(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) object.
+There is no need to explicitly enable Istio authorization feature, you just apply
+the `AuthorizationPolicy` on **workloads** to enforce the access control.
 
-In the `ClusterRbacConfig` object, the operator can specify a `mode` value, which can
-be:
+If no `AuthorizationPolicy` applies to a workload, no access control will be enforced,
+In other words, all requests will be allowed.
 
-- **`OFF`**: Istio authorization is disabled.
-- **`ON`**: Istio authorization is enabled for all services in the mesh.
-- **`ON_WITH_INCLUSION`**: Istio authorization is enabled only for services and
-  namespaces specified in the `inclusion` field.
-- **`ON_WITH_EXCLUSION`**: Istio authorization is enabled for all services in
-  the mesh except the services and namespaces specified in the `exclusion`
-  field.
+If any `AuthorizationPolicy` applies to a workload, access to that workload is
+denied by default, unless explicitly allowed by a rule declared in the policy.
 
-In the following example, Istio authorization is enabled for the `default`
-namespace.
-
-{{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ClusterRbacConfig
-metadata:
-  name: default
-spec:
-  mode: 'ON_WITH_INCLUSION'
-  inclusion:
-    namespaces: ["default"]
-{{< /text >}}
+Currently `AuthorizationPolicy` only supports "ALLOW" action. This means that if
+multiple authorization policies apply to the same workload, the effect is additive.
 
 ### Authorization policy
 
-To configure an Istio authorization policy, you specify a `ServiceRole` and
-`ServiceRoleBinding`. Like other Istio configuration objects, they are
-defined as
-Kubernetes `CustomResourceDefinition` [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) objects.
-
-- **`ServiceRole`** defines a group of permissions to access services.
-- **`ServiceRoleBinding`** grants a `ServiceRole` to particular subjects, such
-  as a user, a group, or a service.
+To configure an Istio authorization policy, you specify the [`AuthorizationPolicy` configuration object](
+/docs/reference/config/authorization/authorization-policy/). Like other Istio configuration objects, it
+is defined as Kubernetes `CustomResourceDefinition` [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+object.
 
-The combination of `ServiceRole` and `ServiceRoleBinding` specifies: **who** is
-allowed to do **what** under **which conditions**. Specifically:
+An `AuthorizationPolicy` includes a `selector` and a list of `rules`, the `selector`
```

Comment:
An `AuthorizationPolicy` includes a `selector` and a list of `rules` ~,~ . ~t~ The `selector`